carcara/src/ast/printer.rs

```diff
@@ -117,13 +123,13 @@ impl<'a> PrintProof for AlethePrinter<'a> {
         while let Some(command) = iter.next() {
             match command {
                 ProofCommand::Assume { id, term } => {
-                    write!(self.inner, "(assume {} ", quote_symbol(id))?;
+                    write!(self.inner, "(assume {} ", id)?;
```

Comment:
You accidentally reverted commit b9cf66b4b664b955e8dbd4fe4ae1b1d9fe736b48. Probably an error when rebasing.